docs/operate-and-deploy/installation/server-config/security.md

```diff
@@ -156,22 +150,31 @@ KsqlServer-Props {
 };
 ```
 
-The example `jaas_config.file` above uses the Jetty
-`PropertyFileLoginModule`, which itself authenticates users by checking
-for their credentials in a password file.
-
-Assign the `KsqlServer-Props` section to the `authentication.realm`
-config setting:
+In this example, the `authentication.realm` config should be set to `KsqlServer-Props`:
 
 ```properties
 authentication.realm=KsqlServer-Props
 ```
 
+The example `jaas_config.file` above uses the Jetty
+`PropertyFileLoginModule`, which itself authenticates users by checking
+for their credentials in a password file.
+
 You can also use other implementations of the standard Java
 `LoginModule` interface, such as `JDBCLoginModule` for reading
 credentials from a database or the `LdapLoginModule`.
 
-The file parameter is the location of the password file. The format is:
+The JAAS config file should be passed as a JVM option during server start,
+and the LoginModule implementation should provided to the server as well:
+
+```bash
+export KSQL_OPTS=-Djava.security.auth.login.config=/path/to/the/jaas_config.file
+export KSQL_CLASSPATH=<path-to-login-module-jar>/jetty-jaas-9.4.24.v20191120.jar
```

Comment:
Do we think it's worthwhile to also show what this looks like via docker? It might not be clear to some users that these options should be passed in the `environment` section, and how to mount the jar as a volume.

This relates to the larger issue that our recommended deployment model for ksqlDB is docker yet many of docs are written without docker as the focus. It's not clear to me whether we should rewrite the docs to be docker-focused, or to have two versions (docker and non-docker) for examples such as this one.

Reply:
I'm always eager to keep parity in the Docker content. I tried to factor out the non-Docker content from docs.ksqldb.io, but there are still some islands of CP-ish content remaining.

Reply:
Not sure I follow. True parity would mean having two versions of each example in the docs, one for Docker and one for non-Docker, right? Having a separate page that acts as a cheatsheet for "how to do the equivalent things (set properties, load UDFs, connectors, jars, etc) in Docker" seems like a good short-term solution so Docker users can make sense of the existing docs with non-Docker examples, but it doesn't seem great in the long run, especially since we recommend installing via Docker.

Reply:
> Maybe we could add this somewhere in here: https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/install-ksqldb-with-docker/

I see the page already has a section on setting system properties, so I assume you're suggesting we add a note about `KSQL_CLASSPATH` into that section, and add a new section on how to mount volumes (for UDFs, connectors and other custom JARs)? If so, SGTM!

Reply:
@vcrfxia We do need to keep two separate docs sets, one for ksqlDB Standalone and the other for ksqlDB Enterprise. Ideally, we'll have Docker-only workflows on docs.ksqldb.io and CP-only workflows on docs.confluent.io, but it would be good to have a mapping between the two somewhere. Currently, we have a very clumsy duct-tape solution that we need to fix.

Reply:
Makes sense. Thanks for the context!

Shall we file tickets to track the improvements you mentioned (adding a note about `KSQL_CLASSPATH` into that docker system properties section, and adding a new section on how to mount volumes (for UDFs, connectors and other custom JARs))? I can file them if you're swamped.

Reply:
That would be great, thanks!